tests/system/datastore/test_v3io.py

```diff
@@ -89,13 +89,15 @@ def _setup_df_dir(self, first_file_path, second_file_path, file_extension):
     @pytest.mark.skip(
         reason="Skipping this test as it hangs when running against the CI system. ML-5598"
     )
-    def test_v3io_large_object_upload(self, tmp_path):
+    @pytest.mark.parametrize(
+        "file_size", [4 * 1024 * 1024, 20 * 1024 * 1024]
```

Comment:
Why those sizes? See no logic. Maybe 99Kb and 10Mb since the max_chunk_size is 100Kb?

Reply:
4MB is in the range of this bug (as I mentioned in the pr description).
99Kb and 100MB or not in this range.

20MB was the previous size of the file of this test.